### PR TITLE
Nexus Mods presence.

### PR DESCRIPTION
**Pull request by Senexis**
This presence is for Nexus Mods. A very popular game modding site. It aims to implement most of the pages on Nexus Mods, and takes some sensitive pages into account while still displaying status on those pages thanks to a generic status.

image
image
image
image
image
image
image

Possibly sensitive page examples:
image
image

### CODE REVIEW

File: NexusMods/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "author": {
+    "name": "Bob Ross",
+    "id": "104726244964196352"
+  },
+  "url": [
+    "nexusmods.com"
+  ],
```
Comment (Bas950) [inline NexusMods/dist/metadata.json]: url shouldnt be in an array if its only 1

File: NexusMods/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "description": {
+    "en": "We support modding for all PC games. If you can mod it, we'll host it."
+  },
+  "service": "Nexus Mods",
+  "version": "1.0.0",
```
Comment (Bas950) [inline NexusMods/dist/metadata.json]: Service name and folder name should be the same, spaces included

File: Nexus Mods/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,257 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+  return new (P || (P = Promise))(function (resolve, reject) {
+    function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+    function rejected(value) { try { step(generator['throw'](value)); } catch (e) { reject(e); } }
+    function step(result) { result.done ? resolve(result.value) : new P(function (resolve) { resolve(result.value); }).then(fulfilled, rejected); }
+    step((generator = generator.apply(thisArg, _arguments || [])).next());
+  });
+};
```
Comment (doomlerd) [inline Nexus Mods/dist/presence.js]: Please remove this, your file is not compiled of TypeScript

File: Nexus Mods/dist/presence.js [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+
+var presence = new Presence({
+  clientId: '683438933841018928',
+  mediaKeys: false
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on('UpdateData', () => __awaiter(this, void 0, void 0, function* () {
```
Comment (doomlerd) [inline Nexus Mods/dist/presence.js]: Change this as well. ( there are examples in the docs )

File: Nexus Mods/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  "tags": [
+    "nexus",
+    "mods",
+    "modding",
+    "nexusmods",
+    "vortex",
+    "modmanager",
+    "nexusmodmanager"
+  ],
+  "category": "game",
```
Comment (Alanexei) [inline Nexus Mods/dist/metadata.json]: Needs to be `games`

Comment (Senexis) [reply Nexus Mods/dist/metadata.json]: Will fix. Should probably update the json generator to reflect this as well, seems to generate with `game`.

File: Nexus Mods/dist/presence.js (hunk 3 above)
Comment (Senexis) [reply Nexus Mods/dist/presence.js]: On it, taken from other presences that use JS. Did the guidelines for that change?

**Comment by Senexis** [comment]
image
Still working fine.

File: Nexus Mods/dist/presence.js (hunk 3 above)
Comment (doomlerd) [reply Nexus Mods/dist/presence.js]: Not specifically, but it's source of issues.

**Pull request merged**